packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/create-remote-media-item-node.js

```diff
@@ -255,110 +256,138 @@ export const createRemoteMediaItemNode = async ({
   const fetchState = {
     shouldBail: false,
   }
-  // Otherwise we need to download it
-  const remoteFileNode = await retry(
-    async () => {
-      if (fetchState.shouldBail) {
-        failedImageUrls.add(mediaItemUrl)
-        return null
-      }
 
-      const createFileNodeRequirements = {
-        parentNodeId: mediaItemNode.id,
-        store: gatsbyStore,
-        cache,
-        createNode,
-        createNodeId,
-        reporter,
-      }
+  const createFileNodeRequirements = {
+    parentNodeId: mediaItemNode.id,
+    store: gatsbyStore,
+    cache,
+    createNode,
+    createNodeId,
+    reporter,
+  }
 
-      if (hardCacheMediaFiles) {
-        // check for file in .wordpress-cache/wp-content
-        // if it exists, use that to create a node from instead of
-        // fetching from wp
-        try {
-          const buffer = await fs.readFile(hardCachedFilePath)
-          const node = await createFileNodeFromBuffer({
-            buffer,
-            name: title,
-            ext: path.extname(mediaItemUrl),
-            ...createFileNodeRequirements,
-          })
+  let imageNode
```

Comment:
Is this only ever going to be used with images? Or will it also be used with other media/file types? If so, maybe we should modify the variable name here to account for those cases.

Reply:
That's a good Q and I'm not 100% sure on that actually. @TylerBarnes would probably know for sure off the top of his head.

Regardless, it does feel a little inconsistent. I'll update it :smile: